tests/TestingTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Tempest;
+
+class TestingTest extends \Tempest\Testing\TestCase
```

Comment:
Isn't this a bit of a weird name?

Reply:
Yes, this goes away in the last commit. This is just my dev playground file. 😛